datalad/interface/__init__.py

```diff
@@ -84,6 +84,16 @@
         ('datalad.distribution.subdatasets', 'Subdatasets', 'subdatasets'),
     ])
 
+# Avoid _group_revolution to prevent confusing interface commands.
+_group_future = (
+    'Commands of the future',
```

Comment:
I guess that should change prior whatever release is coming after an initial consolidation period.

Reply:
Yeah, I think so.  And I'm not even sure that is a good way to refer to it _for now_, but nothing better came to mind.